managed/models/database.go

```diff
@@ -1150,7 +1150,19 @@ var databaseSchema = [][]string{
 		`UPDATE agents SET disabled = true WHERE agent_type = 'qan-postgresql-pgstatements-agent' AND service_id = (SELECT service_id FROM services WHERE service_name = 'pmm-server-postgresql' LIMIT 1);`,
 	},
 	113: {
-		// Reset product tour for new navigation
+		`ALTER TABLE user_flags
+			ADD COLUMN snoozed_at TIMESTAMP,
+			ADD COLUMN snooze_count INTEGER NOT NULL DEFAULT 0`,
+	},
+	// Default snooze duration - 7 days
+	114: {
+		`UPDATE settings
+			SET settings = settings || '{"updates": {"snooze_duration": 604800000000000}}'
```

Comment:
Maybe could you assign duration value into variable? Like weekDuration or something?